Failed to complete chroot setup.

I tried changing from xfce4 to gnome, still won't work. Whats wrong? I've installed ChruBuntu before.

Log:
Compiling CRAS...
In file included from libcras/cras_client.c:39:0:
libcras/cras_client.h:16:29: fatal error: cras_iodev_info.h: No such file or directory
compilation terminated.
alsa_plugin/pcm_cras.c:8:25: fatal error: cras_client.h: No such file or directory
compilation terminated.
alsa_plugin/ctl_cras.c:8:25: fatal error: cras_client.h: No such file or directory
compilation terminated.
tests/cras_test_client.c:14:25: fatal error: cras_client.h: No such file or directory
compilation terminated.
/usr/lib/gcc/i686-linux-gnu/4.6/../../../i386-linux-gnu/crt1.o: In function `_start':
(.text+0x18): undefined reference to`main'
collect2: ld returned 1 exit status
Failed to complete chroot setup.
Unmounting /usr/local/chroots/precise...

--- a/targets/audio
+++ b/targets/audio
@@ -81,6 +81,7 @@ addtrap "rm -rf '$CRASBUILDTMP' 2>/dev/null"
     {
         echo '
             top_srcdir=".."
+            top_builddir=".."
             SBC_LIBS="'"`pkg-config --libs sbc`"'"
             SBC_CFLAGS="'"`pkg-config --cflags sbc`"'"
             DBUS_LIBS="'"`pkg-config --libs dbus-1`"'"

